packages/core-sass/README.md

```diff
@@ -0,0 +1,31 @@
+# hds-core-sass
+
+SASS library for the Helsinki Design System.
+
+## Getting started
+
+### What's included
+
+```
+hds-core-sass/
+├── src/                # source scss
+    └── stylesheets/
+    |   ├── base/
+    |   │   ├── _reset.scss
+    |   |   ├── _typography.scss
+    |   │   ├── <component name>.css
+    |   │   :
+    |   ├── utilities/
+    |   |    ├── _color_variables.scss
+    |   ├── main.scss
+    |
+    └── icons/          # collection of HDS icon styles
```

Comment:
This should stay under dir name assets i think 
``` assets > icons > svg > **```

Reply:
It was not in the core module